Automattic/wp-calypso
# Stats: Update the Highlights section on the Subscribers page.

Related to #75855.

## Proposed Changes

Replaces the `AnnualHighlightsSection` component (from the Traffic page) with a new `SubscribersHighlightSection` component specific to this page.

- Mostly lines up with the designs in 75855 except for the number formatting. Currently using the same formatting rules as the Traffic page for consistency.
- Not wired to any real data yet. Just using hard-coded data per the initial designs.

Will address the following (as needed) in follow-up PRs:

1. Tooltip content for "i" icon.
2. Do we want tooltips on the individual values? Currently enabled.
3. Do we want icons for each card?


## Testing Instructions



* Open the Live branch.
* Visit Jetpack → Stats.
* Append `?flags=stats/subscribers-section` to the URL and load.
* Visit the new Subscribers page.
* Confirm the new "All time stats" section is visible and behaves as expected.

<img width="1250" alt="image" src="https://user-images.githubusercontent.com/4044428/234363756-4189f0a9-89de-4124-8f46-845c43366765.png">

## Pre-merge Checklist
